upgrade/scripts/upgrade/ncn-upgrade-worker-nodes.sh

```diff
@@ -34,10 +34,15 @@ jsonArray=$(jq -r --compact-output --null-input '$ARGS.positional' --args -- "${
 
 ${basedir}/../../../workflows/scripts/upload-worker-rebuild-templates.sh
 
+if [[ -z "${SW_PASSWORD}" ]]; then
+  read -r -s -p "Switch password:" SW_PASSWORD
+fi
+
 cat << EOF > data.json
 {
   "dryRun": false,
-  "hosts": ${jsonArray}
+  "hosts": ${jsonArray},
+  "switchPassword": "${SW_PASSWORD}"
```

Comment:
it would be nice if we have k8s secret manages switch password, then i can just mount it into argo so we don't have to deal with plain text password lol